qiskit/aqua/components/variational_forms/ry.py

```diff
@@ -138,17 +155,33 @@ def construct_circuit(self, parameters, q=None):
 
         for block in range(self._depth):
             circuit.barrier(q)
+            if self._entanglement == 'circular':
+                self._entangler_map = VariationalForm.get_entangler_map(
+                    self._entanglement,
+                    self._num_qubits,
+                    offset=block)
+
             for src, targ in self._entangler_map:
-                if self._entanglement_gate == 'cz':
+                if self._entanglement_gate.lower() == 'cz':
```

Comment:
I think `lower()` is not needed since the schema checking will raise an error if you do not provide the lower case of `cz`, `cx` and `crx`.

Reply:
Ah yes, I'll change it, thanks!